raw_package/static/css/esphome-2.css

```diff
@@ -41,7 +41,7 @@ body {
 
 /* Layout */
 main {
-  margin-top: 64px;
+  margin-top: 56px;
```

Comment:
does this mean all the content is 8px closer to the header. Do we compensate that with a padding in the content ?

Reply:
The padding isn't needed. The main element has space on the top (above the cards). I'll add screenshot showing before and after.

Reply:
I think you forgot to add the screenshot?

Reply:
I'm away from my pc, will add today when I'm home.